Homework2/StackCalculator/TestStackCalculator/TestStackCalculator.cs

```diff
@@ -0,0 +1,82 @@
+using NUnit.Framework;
+using StackCalculator;
+using System;
+
+namespace TestStackCalculator;
+
+/// <summary>
+/// A class for testing a stack calculator
+/// </summary>
+public class TestsStackCalculator
+{
+    Calculator? stackCalculator;
```

Comment:
```suggestion
    private Calculator? stackCalculator;
```
В сообществе принято писать модификаторы видимости всегда, даже если умолчания устраивают